paper.md

```diff
@@ -194,24 +196,23 @@ With this simple approach, git-annex enables separate and optimized implementati
+Beyond scalability, modularization into homogeneous components also enables efficient reuse of a selected subset of datasets and for recording a derivative relationship between datasets.
+Git's *submodule* mechanism provides a mechanism to nest individual repositories via unambiguously versioned linkage, but Git operations must still be performed within each individual repository.
+To achieve modularity without impeding usability, DataLad simplifies working with the resulting hierarchies of Git repositories via recursive operations across dataset boundaries.
+With this, DataLad provides a "mono-repo"-like user experience in datasets with arbitrarily deep nesting, and makes it trivial to operate on individual files deep in the hierarchy or entire trees of datasets.
 A testament of this is [datasets.datalad.org](http://datasets.datalad.org), created as the project's initial goal to provide a data distribution with unified access to already available public data archives in neuroscience, such as [crcns.org](http://crcns.org) and [openfmri.org](http://openfmri.org).
 It is curated by the DataLad team, and provides, at the time of publication, streamlined access to over 250 TBs of data across a wide range of projects and archives in a fully modularized way.
 
-**"Re-executable" annotation of changes.**
-Digital provenance is crucial for the trustworthiness and reproducibility of a research result, and contributes to the principle of "reusability" of the FAIR principles [@FAIR2016].
-Git captures provenance with annotations of file changes, where changes are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and a Git commit message (a freeform, human-readable text description of the introduced changes).
-Unlike changes to text documents or source code, which are typically done "manually", data manipulations are most often performed by software.
-DataLad exploits this fact and enables automated annotation of changes which result from running an external command.
-DataLad creates a commit message which does not only include a human-readable summary, but also a human- and machine-readable record of the command invocation which introduced the changes.
-This allows for the data "change" to be re-executed to either verify that results reproduce, or to apply such a "change" to a completely different state, irrespective of the quality of the human-made commit message.
+**Re-executable annotation of changes.**
+Digital provenance is crucial for the trustworthiness and reproducibility of a research result, and contributes to the reusability aspect of the FAIR principles [@FAIR2016].
+Git captures provenance with annotations of changes that are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and a commit message (a freeform, human-readable text description of the introduced changes).
```

Comment:
Not really about this PR, but this sentence still is confusing in its structure. How about this?

```suggestion
Git captures provenance with annotations of changes, where changes are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and annotations by a commit message (a freeform, human-readable text description of the introduced changes).
```

Reply:
The sentence is (without parenthesis):

> Git captures provenance with annotations of changes that are typically represented by a patch, and a commit message.

I think your proposal is more correct, but arguably less attractive ;-)

What about

```suggestion
Git captures provenance by annotating a patch (an exact difference between two versions that could be applied to another version of the text file), with a commit message (a freeform, human-readable text description of the introduced changes).
```

Reply:
Yes, better. :)